hack/install-tools.sh

```diff
@@ -1,13 +1,12 @@
 #!/bin/bash
 
-set -e
+set -o errexit
+set -o nounset
+set -o pipefail
 
 STARTTIME=$(date +%s)
```

Comment:
Right, since the exit handler for this script doesn't make use of the var, we can remove it. The way this happens is changing soon anyway.